### PR TITLE
update to purescript 0.15

### PR DESCRIPTION
**Pull request by gbagan**
upgrade spago.dhall, bower.json and package.json to latest versions

### CODE REVIEW

**Review by gbagan — COMMENTED** [review]
replace the url git://github.com/anttih/purescript-rationals.git with git://github.com/purescript-contrib/purescript-rationals.git

File: bower.json [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
-    "purescript-integers": "^5.0.0",
-    "purescript-prelude": "^5.0.0"
-  },
-  "devDependencies": {
-    "purescript-bigints": "^5.0.0",
-    "purescript-console": "^5.0.0",
-    "purescript-quickcheck": "^7.0.0",
-    "purescript-quickcheck-laws": "^6.0.0"
-  }
+    "name": "purescript-rationals",
```
Comment (f-f) [inline bower.json]: Could we just apply the changes without reformatting the file?

**Review by gbagan — COMMENTED** [review]
@f-f yes, sorry, I fixed this

**Review by f-f — APPROVED** [review]
Thanks!

**Pull request merged**